ros/src/computing/perception/detection/lidar_tracker/packages/lidar_imm_ukf_pda_track/nodes/imm_ukf_pda/ukf.cpp

```diff
@@ -426,193 +768,227 @@ void UKF::randomMotion(const double p_x, const double p_y, const double v, const
   state[4] = yawd_p;
 }
 
+void UKF::initCovarQs(const double dt, const double yaw)
+{
+  if (tracking_num_ != TrackingState::Init)
+  {
+    return;
+  }
+  double dt_2 = dt * dt;
+  double dt_3 = dt_2 * dt;
+  double dt_4 = dt_3 * dt;
+  double cos_yaw = cos(yaw);
+  double sin_yaw = sin(yaw);
+  double cos_2_yaw = cos(yaw) * cos(yaw);
+  double sin_2_yaw = sin(yaw) * sin(yaw);
```

Comment:
cos(yaw) and sin(yaw) are calculated multiple times. It's better to use variable to save these value.

Reply:
This pr aims to change topic type.
Definitely modifying in next pr.